entity-framework/core/modeling/query-types.md

```diff
@@ -1,53 +1,49 @@
 ---
-title: Query Types - EF Core
-author: anpete
+title: Keyless Entity Types - EF Core
```

Comment:
should rename file.

Reply:
Don't want to deal with changing links/adding a redirect.
@divega thoughts?

Reply:
It makes sense to rename the file, but I think it is a lower priority. If we do rename the file, we need to add a redirection on https://github.com/aspnet/EntityFramework.Docs/blob/master/.openpublishing.redirection.json